packages/expo-modules-autolinking/src/dependencies/utils.ts

```diff
@@ -133,7 +139,18 @@ export async function filterMapResolutionResult<T extends { name: string }>(
   const resolutions = await Promise.all(
     Object.keys(results).map(async (key) => {
       const resolution = results[key];
-      return resolution ? await filterMap(resolution) : null;
+      let result = resolution ? await filterMap(resolution) : null;
```

Comment:
I believe this can be refactored to avoid using `let`

Reply:
`if ((result = ...) != null) return result` is the reason why I asked to refactor it. Both make the code less readable.